examples/init_repo_with_secrets/main.tf

```diff
@@ -36,19 +36,39 @@ module "example" {
     SOME_PLAIN_TEXT_SECRET = {
       plaintext_value = "some_secret"
     }
+
     ENCRYPTED_SECRET = {
       # Value encrypted with organization public key
       # Public key: https://docs.github.com/en/rest/reference/actions#get-an-organization-public-key
       # Ecnryption: https://docs.github.com/en/rest/reference/actions#create-or-update-an-organization-secret
       encrypted_value = "P1wD+Byzy0JvL77qILs1gLj1wpDIDYIKGcHJbuILlTq3lNLgxDQuHXLVYknj2nx6uaeNGx3AmgsO+Nak"
     }
   }
+
+  bot_secrets = {
+    BOT_TEST_SECRET = {},
+
+    BOT_PLAIN_TEXT_SECRET = {
+      bot_plaintext_value = "other_secret"
+    }
+
+    BOT_ENCRYPTED_SECRET = {
+      # Value encrypted with organization public key
+      # Public key: https://docs.github.com/en/rest/reference/actions#get-an-organization-public-key
+      # Ecnryption: https://docs.github.com/en/rest/reference/actions#create-or-update-an-organization-secret
+      bot_encrypted_value = "P1wD+Byzy0JvL77qILs1gLj1wpDIDYIKGcHJbuILlTq3lNLgxDQuHXLVYknj2nx6uaeNGx3AmgsO+Nak"
```

Comment:
Why don't we use sops like for the other secrets?

Reply:
In our private repo, we use sops there. And I think it's ok to present idea how we encrypt secrets in example code.